src/overview/reducer.js

```diff
@@ -5,17 +5,23 @@ import * as actions from './actions'
 const defaultState = {
     searchResult: {rows: []},
     query: '',
+    isFetching:0,
 }
 
 function setQuery(state, {query}) {
     return {...state, query}
 }
 
 function setSearchResult(state, {searchResult}) {
-    return {...state, searchResult}
+    return {...state, searchResult, isFetching:1}
+}
+
+function playLoading(state){
+	return {...state, isFetching:0}
```

Comment:
Should this not be set to 1 or `true` here? Looks like it is the wrong way around.